x11-wm/notion/notion-3_p2015061300.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/x11-wm/notion/notion-3_p2013030200.ebuild,v 1.4 2014/05/25 13:01:33 jer Exp $
+
+EAPI=5
+
+inherit eutils multilib toolchain-funcs
+
+DESCRIPTION="Notion is a tiling, tabbed window manager for the X window system"
+HOMEPAGE="http://notion.sourceforge.net"
+SRC_URI="https://github.com/raboof/${PN}/archive/${PV/_p/-}.tar.gz"
+
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="nls xinerama +xrandr"
+
+RDEPEND=">=dev-lang/lua-5.1
+	x11-libs/libSM
+	x11-libs/libX11
+	x11-libs/libXext
+	nls? ( sys-devel/gettext )
```

Comment:
according to the readme.md it requires gettext. https://github.com/raboof/notion